x/commerciokyc/spec/01_state.md

```diff
@@ -4,9 +4,70 @@ order: 1
 
 # State
 
-The `x/commerciokyc` module keeps state of the follow objects
+The `commerciokyc` module keeps state of the follow objects
 
-- Memberships: `-`
-- Invites: `-`
-- ABR liquidity pool: `-`
+## Store
 
+### Memberships
+
+
+| Key |  | Value |
+| ------- | ---------- | ---------- | 
+| `commerciokyc:storage:[owner]` | &rarr; | _Membership_ |
```

Comment:
in the review of `did` documentation, @SerigneCheikh98 suggested that the arrow is redundant in this kind of table. Thus, I removed it from the `did` documentation. We should decide if we want to remove the arrow from all these documentation files, too. @marcotradenet

Reply:
Leave arrows. Thanks